### DISCUSSION TITLE
What git merge strategy do you use?

### DISCUSSION THREAD

**Discussion by husk-9** [BryanJenksCommunity/FAQ#32] [Q&A] [answered]
Hi Bryan, found your YouTube videos and really enjoyed them! I started using Obsidian yesterday!

So I watched your video about version control and read the [article](https://medium.com/analytics-vidhya/how-i-put-my-mind-under-version-control-24caea37b8a5) too, but I had one question: **What merging strategy do you use**?

If I write a note and push it, then edit my notes on a different computer, I can `git pull` them on my main. However, if I run into a situation where I end up editing notes on the same time on two different devices, with the regular merging, `git pull` will create a merge commit, and it opens Vim to write the message. So how does this work when git is in quiet mode? Do you set [auto merge to true](https://stackoverflow.com/questions/11744081/why-is-git-prompting-me-for-a-post-pull-merge-commit-message/41903855)?

**Comment by tallguyjenks** [comment, marked as answer]
The way the workflow works for me and the way i intended it to work is that the whole operation is for when you leave your personal computer at home and you're on another computer elsewhere. in which case you're only editing on one computer at a time. I may have mentioned it in the video/article that is isn't a solution for a collab/team environment, and doesn't play well in your given scenario. :)
